WORKSPACE.bazel

```diff
@@ -358,10 +358,10 @@ versioned_http_archive(
 versioned_http_archive(
     name = "zlib",
     build_file = "//bazel/third_party/zlib:zlib.BUILD",
-    sha256 = "b3a24de97a8fdbc835b9833169501030b8977031bcb54b3b3ac13740f846ab30",
+    sha256 = "ff0ba4c292013dbc27530b3a81e1f9a813cd39de01ca5e0f8bf355702efa593e",
     strip_prefix = "zlib-{version}",
     url = "https://www.zlib.net/zlib-{version}.tar.gz",
-    version = "1.2.13",
+    version = "1.3",
```

Comment:
I've seen this changed in at least a couple PRs. @matthew-levan can you follow up with this dependency update?

Reply:
I think @matthew-levan fixed this already in `develop`, but I needed to cherry-pick the commit because I'm working from `vere-v2.11` (I couldn't find a working bootstrap pill for `2.12`).